TACC/Frontera-Portal
# bug/FP-820 - Show loading spinner during iframe load

## Overview: ##

This removes the "no spinner but no preview" condition in the preview modal

## Related Jira tickets: ##

* [FP-824](https://jira.tacc.utexas.edu/browse/FP-824)

## Summary of Changes: ##

The preview iframe has a callback to that hides the loading spinner after it has finished loading.

## Testing Steps: ##
1. Save an image to your My Data
2. Click on the image to open the preview modal
3. You should see that the spinner stays up while the browser status in the lower left of the window shows that it is waiting for a call to complete

## UI Photos:

## Notes: ##
